addons/project/models/project_task.py

```diff
@@ -2057,3 +2057,33 @@ def get_mention_suggestions(self, search, limit=8):
         ])
         partners = self.env["res.partner"].sudo()._search_mention_suggestions(domain, limit)
         return Store(partners).get_result()
+
+    # ----------------------------------------------------
+    # Gantt view
+    # ----------------------------------------------------
+
+    @api.model
+    def get_gantt_data(self, domain, groupby, read_specification, limit=None, offset=0, unavailability_fields=[], progress_bar_fields=None, start_date=None, stop_date=None, scale=None):
+        res = super().get_gantt_data(domain, groupby, read_specification, limit=limit, offset=offset, unavailability_fields=unavailability_fields, progress_bar_fields=progress_bar_fields, start_date=start_date, stop_date=stop_date, scale=scale)
+
+        progress_bars = res.get('progress_bars')
+
+        if not progress_bars:
+            return res
+
+        last_group_by = progress_bar_fields[-1]
+        progress_bars['__default'] = {
```

Comment:
I would add a comment here to state that this is for the total row's progress bar - to make it a bit clearer 
(This applies to the enterprise PR as well)